lib/gateway/Shard.js

```diff
@@ -31,6 +31,9 @@ var Inflator = typeof window !== "undefined" && Pako ? Pako.inflate : Zlib.infla
 * @prop {Number} guildCount The number of guilds this shard should be handling
 * @prop {Array<String>?} discordServerTrace Debug trace of Discord servers
 * @prop {String} status The status of the shard. "disconnected"/"connecting"/"handshaking"/"connected"
+* @prop {Number} lastHeartbeatReceived Timestamp when Discord last acknowledged a heartbeat
+* @prop {Number} lastHeartbeatSent Timestamp when shard last sent a heartbeat
```

Comment:
Yeah, I thought it was, but I couldn't figure out a better way to word it. Ideas?